src/ga4gh/vrs/extras/translator.py

```diff
@@ -175,11 +189,16 @@ def _from_gnomad(self, gnomad_expr, assembly_name=None):
             assembly_name = self.default_assembly_name
         sequence_id = assembly_name + ":" + g["chr"]
         start = int(g["pos"]) - 1
-        ref = g["ref"]
-        alt = g["alt"]
+        ref = g["ref"].upper()
+        alt = g["alt"].upper()
         end = start + len(ref)
         ins_seq = alt
 
+        # validation checks
+        valid_ref_seq, err_msg = self._is_valid_ref_seq(sequence_id, start, end, ref)
+        if not valid_ref_seq and require_validation:
+            raise ValidationError(err_msg)
```

Comment:
Do we want to enter this section at all if `require_validation` is false?

Reply:
I'll switch the order!